src/main/java/frc/robot/Constants.java

```diff
@@ -57,4 +57,13 @@ public class Controller {
         public static final int LEFT_JOYSTICK_BUTTON = 9;
         public static final int RIGHT_JOYSTICK_BUTTON = 10;
     }
+
+    public class Undertaker {
+        public static final int INTAKE_MOTOR_ONE = 13;
+        public static final int INTAKE_MOTOR_TWO = 14;
+
+        public static final double INTAKE_VOLTAGE = 6.0;
```

Comment:
after discussion with @python-42 

let's change this back to `INTAKE_SPEED` and be -1 to 1 (so this would be 0.5). Then multiply by 12 in the `Undertaker` subsystem method

Same with eject